package.json

```diff
@@ -115,7 +116,7 @@
     "source-map-loader": "^0.2.4",
     "storybook-addon-performance": "^0.9.0",
     "ts-node": "^8.8.2",
-    "typescript": "^3.9.5",
+    "typescript": "^4.0.1-rc",
```

Comment:
using 4 so that `"noEmit": true` can be used together with `"incremental": true` (implied by `"composite": true`)